results/beck_new.py

```diff
@@ -881,6 +879,263 @@ def tm(x):
 
         self.save()
         tm('save')
+        Report(self.data, self.dir).make()
+
+
+class Report:
```

Comment:
There are two `Report` classes now - we should remove or rename one of them.

Reply:
Some trouble after webeditor dev merging